### DISCUSSION TITLE
Uploading script to github

### DISCUSSION THREAD

**Discussion by acwimbley** [community/community#77527] [New to GitHub]
I am taking a class in which we are to upload our VBA scripts in github..  I am new at this.  The repository is already created.

**Comment by guspan-tanadi** [comment]
In the [repository](https://github.com/acwimbley/VBA-Challenge) choose **Add file** then **Upload files** as referred in GitHub Docs [Add a file](https://docs.github.com/en/repositories/working-with-files/managing-files/adding-a-file-to-a-repository#adding-a-file-to-a-repository-on-github).

Next click choose your files link a dialog will appear for the user to find folder location. Next phase simply drag the folder or file as described in graphic below. Look for uploading process. Fill the input in Commit changes form for example "Add VBA script". Confirm by click Commit changes button.

![a888b59e-d004-11e5-80a6-e9f32f17c4d5](https://github.com/community/community/assets/36249910/1799688f-f5ee-48a9-a95a-fc14068487d0)
Credit for picture was taken from [Aaron Shekey](https://github.blog/author/aaronshekey) [posts](https://github.blog/2016-02-18-upload-files-to-your-repositories) on [The GitHub Blog](https://github.blog)

There the script has been uploaded to repo.
Example of [VBA Projects](https://github.com/Slayzerus/Excel-VBA-Projects) is also provided which has file extension in upload as [`vbs`](https://github.com/Slayzerus/Excel-VBA-Projects/blob/code/SequenceSystem%20(SAP)/194a.vbs) extension.

In the meanwhile, consider an option of [command line](https://docs.github.com/en/repositories/working-with-files/managing-files/adding-a-file-to-a-repository#adding-a-file-to-a-repository-using-the-command-line) usage.

**Reply by acwimbley** [reply]
Thank you for your prompt response.  I see that I did not explain my situation clearly and will try again.  I need to create script files from the macros from what I created in an Excel file.  Then I can follow your explanation on how to upload them.  Thanks and sorry for not being clear in the first place -- a newbie

**Reply by guspan-tanadi** [reply]
So, the actual discussion is regarding create VBA Macro in Spreadsheet Excel rather than uploading script there is? If so can have another discussion then.

**Comment by acwimbley** [comment]
Thank you again.  I was able to get assistance on this and now I can help my fellow classmates.   I didn't understand exporting in VBA and now I do.  Thanks again for getting back to me on this.
    On Friday, December 1, 2023 at 03:56:52 PM CST, Guspan Tanadi ***@***.***> wrote:  
 
 


So, the actual discussion is regarding create VBA Macro in Spreadsheet Excel rather than uploading script there is? If so can have another discussion then.
